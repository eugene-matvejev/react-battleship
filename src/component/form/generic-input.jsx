import React from 'react';
import composeInput from './compose-input';

const GenericInput = (props) => <input {...props} />;

GenericInput.propTypes = {
};
GenericInput.defaultProps = {
};

export default composeInput(GenericInput);
export {
    GenericInput as PureGenericInput
}
