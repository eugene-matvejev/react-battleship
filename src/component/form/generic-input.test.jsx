import React from 'react';
import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import { PureGenericInput } from './generic-input';

configure({ adapter: new Adapter() });

describe('<GenericInput/>', () => {
    const props = {
    };

    describe('render', () => {
        it('with default/required props', () => {
            const c = shallow(<PureGenericInput {...props} />);

            expect(c).toMatchSnapshot();
        });
    });
});
