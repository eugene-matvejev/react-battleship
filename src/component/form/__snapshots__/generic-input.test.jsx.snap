// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<GenericInput/> render with default/required props 1`] = `<input />`;
